### PR TITLE
Makes our "connect" script work. A bit of abuse to pyotp.

### PR DESCRIPTION
**Pull request by cobbr2**
And this is the actual change that made connect work for me; I'm confident I'm abusing the pyotp interface, though, so Gabe you may want to look closely.

### CODE REVIEW

File: gp-okta.conf [hunk 1]
```diff
@@ -2,15 +2,15 @@
 debug = 0
 vpn_url = https://grandrounds.gpcloudservice.com 
 okta_url = https://grandrounds.okta.com
-username = marcelo.vanzin
+username = rick.cobb
```
Comment (gvacaliuc) [inline gp-okta.conf]: can provide this via `GP_USERNAME=rick.cobb`

File: gp-okta.conf [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 # password = mypass
 # mfa_order = totp sms
 # sms.okta = 0
 # totp.okta = ABCDEFGHIJKLMNOP
 # totp.google = ABCDEFGHIJKLMNOP
 gateway = US East
 openconnect_cmd = sudo openconnect
-openconnect_args =                    # optional arguments to openconnect
+openconnect_args = --csd-wrapper /usr/lib/x86_64-linux-gnu/openconnect/hipreport.sh
```
Comment (gvacaliuc) [inline gp-okta.conf:13]: i'd prefer this to be set via environment variable and default to a vendored hipreport.sh

e.g. `GP_OPENCONNECT_ARGS="--csd-wrapper $GR_HOME/pan-globalprotect-okta/hipreport.sh"`

**Review by gvacaliuc — COMMENTED** [review]
does gp-okta.py not work for you out of the box using totp?

**Comment by cobbr2** [comment]
Not worth completing. New `bin/vpn` works without modifying upstream `gp-okta.py` at all; no reason to keep going.

**Pull request closed without merging**